### PR TITLE
copyedit_c13

### PR DESCRIPTION
**Pull request by jannes-m**

### CODE REVIEW

**Comment by jannes-m** [comment]
Page 285 of the copyedited pdf: In the header it says "13.4 Tidy the input data". It should state either "13.3 Tidy the input data" or "13.4 Create census rasters". Both headers can be found on page 285, the mixture "13.4 Tidy the input data" is awkward. How can we tell Latex not to do so?

**Comment by Nowosad** [comment]
The similar problem occurs several times in the book, e.g. page 129, 133, 191... 
I have no idea why that happens, but we should work on it after the all other improvements are implemented...

File: 13-location.Rmd [hunk 1]
```diff
@@ -498,7 +498,7 @@ reclass = dropLayer(reclass, "pop")
 ```
 
 In common with other data science projects, data retrieval and 'tidying' have consumed much of the overall workload so far.
-With clean data the final step, calculating a final score by summing up all raster layers, can be accomplished in a single line.
+With clean data the final step --- calculating a final score by summing up all raster layers --- can be accomplished in a single line.
```
Comment (jannes-m) [inline 13-location.Rmd]: @Robinlovelace pls check if the sentence is clear and understandable, the copyeditor suggested to change it to:

`With clean data as the final step, calculating a final score by summing up all raster layers can be accomplished in a single line.`
Which is clearly not, what I meant.

Comment (Robinlovelace) [reply 13-location.Rmd]: Looks good except I think there should be an additional comma after 'With the data clean', and removal of the the word 'up' which I think adds little/nothing to the meaning, e.g.:

```
With clean data, the final step --- calculating a final score by summing all raster layers --- can be accomplished in a single line of code.
```

And `line of code` is clearer than `line`.


 Good use of ---!

**Pull request merged**